packages/sw-lib/src/lib/strategies.js

```diff
@@ -0,0 +1,161 @@
+import {
+  CacheFirst, CacheOnly, NetworkFirst,
+  NetworkOnly, StaleWhileRevalidate,
+} from '../../../sw-runtime-caching/src/index.js';
+import {Plugin as CacheExpirationPlugin} from
+  '../../../sw-cache-expiration/src/index.js';
+import {BroadcastCacheUpdatePlugin} from
+  '../../../sw-broadcast-cache-update/src/index.js';
+import {CacheableResponsePlugin} from
+  '../../../sw-cacheable-response/src/index.js';
+import {RequestWrapper} from '../../../sw-runtime-caching/src/index.js';
+
+/**
+ * This is a simple class used to namespace the supported caching strategies in
+ * sw-lib.
+ *
+ * You would never access this class directly but instead use with
+ * `swlib.strategies.<Strategy Name>`.
+ *
+ * @memberof module:sw-lib
+ */
+class Strategies {
+  /**
+   * A [cache first](https://jakearchibald.com/2014/offline-cookbook/#cache-falling-back-to-network)
+   * run-time caching strategy.
```

Comment:
Great that we're linking out to the equivalents in the offline cookbook. Would you mind switching these to use the /web version (more up to date): https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network